lib/ioutil/log_entry.go

```diff
@@ -18,12 +18,9 @@ import (
 	"time"
 )
 
-const logEntryVersion = 1
-
 // LogEntry attaches a timestamp to a log entry
 // easyjson:json
 type LogEntry struct {
-	Log     string    `json:"log"`
-	Time    time.Time `json:"time"`
-	Version int       `json:"version"`
+	Msg string    `json:"msg"`
+	Ts  time.Time `json:"ts"`
```

Comment:
Field names are fine, I was only worrying about json field names, those take space while logging data.
Maybe we should use int64 to store unix time only? It is cheaper and portable.